src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -116,8 +116,17 @@ def __init__(
         trained_betas: Optional[np.ndarray] = None,
         variance_type: str = "fixed_small",
         clip_sample: bool = True,
-        predict_epsilon: bool = True,
+        prediction_type: str = "epsilon",
+        **kwargs,
     ):
+        message = (
+            "Please make sure to instantiate your scheduler with `prediction_type` instead. E.g. `scheduler ="
+            " DDPMScheduler.from_pretrained(<model_id>, prediction_type='epsilon')`."
+        )
+        predict_epsilon = deprecate("predict_epsilon", "0.10.0", message, take_from=kwargs)
+        if predict_epsilon is not None:
+            prediction_type = "epsilon" if predict_epsilon else "sample"
```

Comment:
This doesn't work with `from_pretrained`:

```Python
ddpm = DDPMScheduler.from_pretrained(
    "hf-internal-testing/tiny-stable-diffusion-torch",
    subfolder="scheduler",
    predict_epsilon=False,
)
assert ddpm.prediction_type == "sample"        # AssertionError
```

The reason is that `from_config` extracts `predict_epsilon` as one `unused_kwargs`: https://github.com/huggingface/diffusers/blob/main/src/diffusers/configuration_utils.py#L195

How could we handle this case?

Reply:
Good catch! Let's correct this in a follow up PR ok?